includes/class-give-background-updater.php

```diff
@@ -1,612 +1,637 @@
 <?php
+
+use Give\Framework\WordPressLibraries\WPBackgroundProcess;
+
 /**
  * Background Updater
  *
  * Uses https://github.com/A5hleyRich/wp-background-processing to handle DB
  * updates in the background.
  *
- * @class    Give_Background_Updater
- * @version  2.0.0
- * @package  Give/Classes
- * @category Class
- * @author   GiveWP
+ * @package    Give/Classes
+ *
+ * @class      Give_Background_Updater
+ * @version    2.0.0
+ * @category   Class
+ * @author     GiveWP
+ *
+ * @deprecated Use src/Framework/WordPressLibraries.
```

Comment:
Can you please add the `@deprecated` tag for WP_Async_Request as well.

Reply:
Resolved: a3400254a6558502ccefe5b845fdc14a7562e57c